modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -14,26 +14,18 @@
       'email_generation_digest',
```

Comment:
Could we also increase the threshold for the `email_generation_digest` queue. We've seen it alert occasionally spiking around the 8 minute mark? I was thinking 60 / 90 minutes.

Also is the `email_generation_immediate` queue now redundant? I can't see it mentioned in the email-alert-api codebase.

Reply:
@1pretz1 I'm going to decline your request to keep the scope of this PR tight. When we do change the thresholds, there's a bit more planning/thought that needs to go into it:

- Set them so that the alert is definitely a problem.
- Explain the thresholds in the docs.
- Ensure the alert can be acted on.

> Also is the email_generation_immediate queue now redundant?

Probably - at least we won't alert on that one!